scipy/stats/_multivariate.py

```diff
@@ -5751,9 +5743,66 @@ def _sample_uniform_direction(dim, size, random_state):
     return samples
 
 
+_dirichlet_mn_doc_default_callparams = """\
+alpha : array_like
+    The concentration parameters. The number of entries determines the
+    dimensionality of the distribution.
+n : array_like
+    The number of trials.
+"""
+
+_dirichlet_mn_doc_frozen_callparams = ""
+
+_dirichlet_mn_doc_frozen_callparams_note = """\
+See class definition for a detailed description of parameters."""
+
+dirichlet_mn_docdict_params = {
+    '_dirichlet_mn_doc_default_callparams': _dirichlet_mn_doc_default_callparams,  # noqa
+    '_doc_random_state': _doc_random_state
+}
+
+dirichlet_mn_docdict_noparams = {
+    '_dirichlet_mn_doc_default_callparams': _dirichlet_mn_doc_frozen_callparams,
+    '_doc_random_state': _doc_random_state
+}
```

Comment:
Both `alpha` and `n` needed to be parameters of the distribution, so we can't reuse `_dirichlet_doc_default_callparams` or the rest of these from `dirichlet`.